features/google/src/model/analyses/ministories/awakening-location-analysis.js

```diff
@@ -80,20 +80,21 @@ export default class AwakeningLocationAnalysis extends RootAnalysis {
         let dateDataObject = {};
         const locationTimeline = googleAccount.placeVisits;
         const activityTimeline = googleAccount.activities;
-        locationTimeline.forEach((location) => {
+        locationTimeline?.forEach((location) => {
```

Comment:
But this might end up with an empty dateDataObject, right?

Reply:
yup in which case there was no data